airflow/providers/pinecone/operators/pinecone.py

```diff
@@ -52,7 +52,7 @@ def __init__(
         *,
         conn_id: str = PineconeHook.default_conn_name,
         index_name: str,
-        input_vectors: list[tuple],
+        input_vectors: list[Vector] | list[tuple] | list[dict],
```

Comment:
Can we explain different `input_vectors` types with examples in the doc string?

Reply:
changed it